docs/docs/rest-api/mesosphere/marathon/api/v2/AppsResource_create.md

```diff
@@ -82,6 +82,16 @@ Here is an example of an application JSON which includes all fields.
             "maxConsecutiveFailures": 3
         },
         {
+            "protocol": "HTTP",
+            "path": "/machinehealth",
+            "gracePeriodSeconds": 3,
+            "intervalSeconds": 10,
+            "overridePort": 3333,
+            "portIndex": 0,
```

Comment:
I think we remove portIndex from here.

Having both `overridePort` and `portIndex` is a bit confusing.